### PR TITLE
[MBL-17852][Student][Teacher] Launch Kaltura external tool

### PR DESCRIPTION
**Pull request by domonkosadam**
Test plan: See ticket

refs: MBL-17852
affects: Student, Teacher
release note: Kaltura videos now can be opened with the Launch External Tool button

## Screenshots

<table>
<tr><th>Before</th><th>After</th></tr>
<tr>
<td></td>
<td></td>
</tr>
</table>

## Checklist

- [ ] Tested in light mode

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Parent Install Page</p>

File: apps/student/src/main/java/com/instructure/student/fragment/LtiLaunchFragment.kt [hunk 1]
```diff
@@ -93,13 +110,16 @@ class LtiLaunchFragment : ParentFragment() {
             return
         }
 
+        Log.d("LtiLaunchFragment", "onResume: $ltiUrl")
```
Comment (kristofnemere) [inline apps/student/src/main/java/com/instructure/student/fragment/LtiLaunchFragment.kt:113]: You left this here

File: apps/student/src/main/java/com/instructure/student/fragment/LtiLaunchFragment.kt [hunk 2]
```diff
@@ -164,6 +184,15 @@ class LtiLaunchFragment : ParentFragment() {
         customTabLaunched = true
     }
 
+    private fun String.replaceWithURLQueryParameter(ifSatisfies: Boolean = true): String {
```
Comment (kristofnemere) [inline apps/student/src/main/java/com/instructure/student/fragment/LtiLaunchFragment.kt:187]: Could move to pandautils and use that from both places

**Review by kristofnemere — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

**Review by tamaskozmer — APPROVED** [review]

**Review by hermannakos — APPROVED** [review]

**Pull request merged**